modules/schematics/src/reducer/index.spec.ts

```diff
@@ -27,8 +27,8 @@ describe('Reducer Schematic', () => {
 
   let appTree: UnitTestTree;
 
-  beforeEach(() => {
-    appTree = createReducers(createWorkspace(schematicRunner, appTree));
+  beforeEach(async () => {
+    appTree = await createWorkspace(schematicRunner, appTree);
```

Comment:
Why was `createReducers` removed?

Reply:
It was by mistake and this is why my test cases were failing, thanks for pointing it out.